config/webhook.yaml

```diff
@@ -34,7 +34,7 @@ spec:
           - podAffinityTerm:
               labelSelector:
                 matchLabels:
-                  app: webhook
+                  role: webhook
```

Comment:
@elfotografo007 Could you fix it in the helm-charts https://github.com/sigstore/helm-charts/blob/main/charts/cosigned/templates/policy-webhook/deployment_policy_webhook.yaml#L48 too ?

Reply:
Sure.

Reply:
I noticed that https://github.com/sigstore/helm-charts/blob/main/charts/cosigned/templates/webhook/deployment_webhook.yaml does not include the `affinity` key. Should I add it too?

Reply:
yes, add it to be consistent. We probably missed it. Thanks